### PR TITLE
Newsletter-238: translate into Chinese

### PR DESCRIPTION
**Pull request by hulatown**

### CODE REVIEW

File: _posts/zh/newsletters/2023-02-15-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+[kogman dilution]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-February/021444.html
+[riard dilution]: https://gist.github.com/ariard/7e509bf2c81ea8049fd0c67978c521af#witness-malleability
+[kogman dilution2]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-February/021459.html
+[o'connor tsm]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-February/021452.html
+[bitcoin inquisition #19]: https://github.com/bitcoin-inquisition/bitcoin/issues/19
+[bitcoinsearch.xyz]: https://bitcoinsearch.xyz/
+[core lightning 23.02rc2]: https://github.com/ElementsProject/lightning/releases/tag/v23.02rc2
+[BTCPay Server 1.7.11]: https://github.com/btcpayserver/btcpayserver/releases/tag/v1.7.11
+[bdk 0.27.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v0.27.0
+[news234 liana]: /en/newsletters/2023/01/18/#liana-wallet-released
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2023-02-15-newsletter.md:124]:
```suggestion
[news234 liana]: /zh/newsletters/2023/01/18/#liana
```

File: _posts/zh/newsletters/2023-02-15-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+- [LDK #2002][] 添加了对自动重新发送最初未成功的 [自动付款][topic spontaneous payments] 的功能。
+
+- [BTCPay Server #4600][] 更新其 [payjoin][topic payjoin] 的实现方案的[硬币选择][topic coin selection] 以尽量避免创建具有不必要输入的交易，特别是在包含多个输入的交易中输入大于任何输出的输入。这不会发生在常规的单一支付者、单一接收者支付中：最大的输入将为支付输出提供足够的支付，并且不会添加额外的输入。这个 PR 的部分灵感来自一篇分析 payjoin 的文章[paper analyzing payjoins][]。
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="5361,5670,5956,851,5697,5960,7171,2002,4541,4600" %}
+[news147 backups]: /en/newsletters/2021/05/05/#closing-lost-channels-with-only-a-bip32-seed
+[cln security.md]: https://github.com/ElementsProject/lightning/blob/master/SECURITY.md
+[news222 musig2]: /en/newsletters/2022/10/19/#musig2-security-vulnerability
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2023-02-15-newsletter.md:105]:
```suggestion
[news222 musig2]: /zh/newsletters/2022/10/19/#musig2
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]
@hulatown thanks for the translation. I noticed a couple zh links that were available to use instead of the en versions

**Comment by hulatown** [comment]
> @hulatown thanks for the translation. I noticed a couple zh links that were available to use instead of the en versions

@bitschmidty Thanks Mike! Fixed. Will pay more attention next time.

**Pull request merged**